docs/entities/nft-events/nft-event.schema.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
@hstove @friedger is this API payload adequate for the uses you have in mind? It could also return the transaction ID, block height, etc if needed.

Reply:
It's not bad, and would be valuable to ship in current form. But it would be much better to include txid, block height, timestamp, etc.

What does `value` return for different types? The value could be any Clarity type. I'm not sure what data we have, but including things like `raw_value`, `value_repr`, etc could be good.

Reply:
I was late to the party :-) I have sent a new PR #507 to streamline clarity value representations